packages/wasm/package.json

```diff
@@ -4,19 +4,34 @@
   "license": "(MIT OR Apache-2.0)",
   "type": "module",
   "scripts": {
-    "clean": "rm -fv wasm/index*",
+    "build": "tsc --build",
+    "clean": "rm -rfv dist wasm package penumbra-zone-wasm-*.tgz",
     "clean:rust": "cargo clean --manifest-path ./crate/Cargo.toml",
-    "compile": "cd crate && wasm-pack build --no-pack --target bundler --out-name index --out-dir ../wasm",
+    "compile": "cd crate && wasm-pack build --no-pack --target bundler --out-name index --out-dir ../wasm && touch ../wasm/.npmignore",
```

Comment:
this `.npmignore` existed previously, and is used to disable `npm pack`'s aggressive ignoring of directory contents due to `wasm-pack`'s autogenerated `.gitignore` (which can't be disabled). now that `clean` just blows away the directory, we have to restore it after compile

Reply:
How does `packages/wasm/wasm/.npmignore` relate to `packages/wasm/.npmignore`? Also, kinda feel like the double wasm dir is kinda confusing.

What purpose does the empty .npmignore serve? Does it make npm not publish the output?

Reply:
the empty npmignore makes npm actually pack the directory, instead of using the wildcard gitignore to ignore the files. this already existed, i just changed the clean behavior at the former line 7 and thus now it needs recreation in some cases.

the higher level ignore is for not packing the tests.

i think the double wasm directory is also a bad choice, but i didn't make that choice in this PR.